common/eth2/src/lighthouse_vc/http_client.rs

```diff
@@ -684,6 +717,38 @@ impl ValidatorClientHttpClient {
 
         self.post(path, &()).await
     }
+
+    /// `GET /eth/v1/validator/{pubkey}/graffiti`
+    pub async fn get_graffiti(
+        &self,
+        pubkey: &PublicKeyBytes,
+    ) -> Result<GetGraffitiResponse, Error> {
+        let url = self.make_graffiti_url(pubkey)?;
+        self.get(url)
+            .await
+            .map(|generic: GenericResponse<GetGraffitiResponse>| generic.data)
+    }
+
+    /// `POST /eth/v1/validator/{pubkey}/graffiti`
+    pub async fn set_graffiti(
+        &self,
+        pubkey: &PublicKeyBytes,
+        graffiti: GraffitiString,
+    ) -> Result<(), Error> {
+        let url = self.make_graffiti_url(pubkey)?;
+        let set_graffiti_request = SetGraffitiRequest {
+            graffiti: Some(graffiti),
+        };
+        self.post(url, &set_graffiti_request).await?;
+        Ok(())
```

Comment:
nit:

```suggestion
        self.post(url, &set_graffiti_request).await
```

Reply:
I noticed a few of these - was this intentional for better readability (being more explicit)? I'd usually prefer conciseness if it doesn't hurt readability too much, but not against this either.

Reply:
yeah looking back at it, I agree with you. i think your suggestion is more idiomatic, I'll go with that